packages/cli/src/environments/sourceControl/sourceControlHelper.ee.ts

```diff
@@ -1,33 +1,70 @@
-import { Container } from 'typedi';
+import Container from 'typedi';
+import { License } from '@/License';
 import { generateKeyPairSync } from 'crypto';
-import sshpk from 'sshpk';
+import type { KeyPair } from './types/keyPair';
 import { constants as fsConstants, mkdirSync, accessSync } from 'fs';
 import { LoggerProxy } from 'n8n-workflow';
-import { License } from '@/License';
-import type { KeyPair } from './types/keyPair';
-import { SOURCE_CONTROL_GIT_KEY_COMMENT } from './constants';
+import {
+	SOURCE_CONTROL_GIT_KEY_COMMENT,
+	SOURCE_CONTROL_TAGS_EXPORT_FILE,
+	SOURCE_CONTROL_VARIABLES_EXPORT_FILE,
+} from './constants';
+import type { SourceControlledFile } from './types/sourceControlledFile';
+import path from 'path';
+
+export function stringContainsExpression(testString: string): boolean {
+	return /^=.*\{\{.*\}\}/.test(testString);
```

Comment:
As far as I know `/^=/` is enough to identify an expression in a string parameter. Plus there could be an expression without any bracketed segment, or an expression with an unterminated bracketed segment.

Reply:
I see! Not familiar with these kinds of expressions `=mysecretvalue`